go/src/koding/kites/kloud/contexthelper/publickeys/publickeys.go

```diff
@@ -17,6 +23,17 @@ type Keys struct {
 	KeyName    string
 }
 
+func (k *Keys) Thumbprint() (string, error) {
```

Comment:
I removed this code, it turned out it's both not valid and not usefull here for now.

> thumbprinting is used for root CAs 

And this is exactly the case for Azure, it turned out you don't simply upload a ssh public key - you need to create a cert out of your private key and upload it to Azure [0]. And then you can use the thumbprint of it as an ID. So for now we do not inject kloud's SSH key, we inject instead a random password.

Injecting random password / kloud's ssh key was only a workaround here. It's not about ssh key pair management, although it'd be good if we did it for all the providers but it's outside of the scope of this PR.

[0] https://support.microsoft.com/en-us/kb/2977336